docs/basics/2d_hist.rst

```diff
@@ -155,4 +134,28 @@ Instead of inputing a number of bins and a range in ``make_2d_hist()``, the bins
 Display 1D distributions
 ------------------------
 
-In development.
\ No newline at end of file
+To plot a 2d histogram with x and y projections:
+
+.. code-block:: python
+
+    from plothist import make_2d_hist, plot_2d_hist_with_projections
+
+    name_x = "variable_0"
+    name_y = "variable_1"
+
+    h = make_2d_hist([df[df[name_x]%2==0][name_x], df[df[name_x]%2==0][name_y]])
+
+    fig, ax_2d, ax_x_projection, ax_y_projection, ax_colorbar = plot_2d_hist_with_projections(
+        h,
+        xlabel="variable_0",
+        ylabel="variable_1",
+        ylabel_x_projection="Entries",
+        colorbar_kwargs={"label": "Entries"},
+        save_as="2d_hist_with_projections.svg",
+    )
+
+    fig.savefig("2d_hist_with_projections.svg", bbox_inches='tight')
```

Comment:
Either remove this line as we use `save_as` in the function, or keep this line and remove the `save_as`